### ISSUE TITLE
Integrate binary push and releases into CI

### ISSUE THREAD

**Issue by leahnp** [samsung-cnct/k2-crash-application#8]
Currently binary + releases are push manually via git-releases tool and Makefile. 

Wait for K2Cli releases to be finished and use that style.

**Comment by coffeepac** [comment]
work already done per @leahnp

**Comment by coffeepac** [comment]
blocked by:  samsung-cnct/k2cli#79

**Comment by coffeepac** [comment]
should be converted to being a solas- style application.
